scripts/lsp-start-plain.el

```diff
@@ -41,7 +41,9 @@
        (package-user-dir (expand-file-name (make-temp-name "lsp-tmp-elpa")
                                            user-emacs-directory))
        (custom-file (expand-file-name "custom.el" package-user-dir))
-       (pkg-list '(lsp-mode lsp-ui yasnippet lsp-java lsp-python-ms lsp-haskell helm-lsp lsp-treemacs dap-mode lsp-origami lsp-dart company flycheck lsp-pyright)))
+       (pkg-list '(lsp-mode lsp-ui yasnippet lsp-java lsp-python-ms lsp-haskell helm-lsp lsp-treemacs dap-mode lsp-origami lsp-dart company flycheck lsp-pyright
+                            ;; modes
+                            rust-mode php-mode scala-mode dart-mode)))
```

Comment:
Would you mind adding clojure-mode 😅 ? I'm not sure it's shipped with emacs

Reply:
Sure, this is why we have this PR. :)